Components/Helpers/MollieStatusConverter.php

```diff
@@ -70,6 +70,10 @@ public function getMollieOrderStatus(Order $order)
                 $targetStatus = PaymentStatus::MOLLIE_PAYMENT_FAILED;
             }
 
+            if ($paymentsResult[PaymentStatus::MOLLIE_PAYMENT_EXPIRED] == $paymentsResult['total']) {
```

Comment:
might be a typesafe condition here?

Reply:
na please not, not going to touch that code for now, will be changed anyway soon
but typesafe already brought some problems in that code *sigh :D